MusicTaste/MusicTaste/Cells/User.m

```diff
@@ -2,7 +2,7 @@
 
 @implementation User
 //Generate name and profile pticture
-@synthesize name,profilePicture;
+@synthesize name,profilePicture,artists;
```

Comment:
now stores artists in User file

Reply:
When we looked at this yesterday in our 1-1 we determined that the User model class was not really needed since you are also using `PFUser`. Are there plans to switch?

Reply:
Already switched! Thank you for the heads up!